pom.xml

```diff
@@ -200,6 +212,12 @@
             <artifactId>spark-swagger</artifactId>
             <version>1.0.0.48</version>
         </dependency>
+        <!-- HTTP client library -->
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
```

Comment:
JDK 11 ships with its own http client class `java.net.http.HttpClient`.

Reply:
I tried using the java `HttpClient`, but can't necessarily get rid of this package because it provides `UriBuilder`. I don't think there's a great alternative to that class yet.